litestar/_openapi/request_body.py

```diff
@@ -48,4 +49,8 @@ def create_request_body(
     else:
         schema = schema_creator.for_field_definition(data_field)
 
-    return RequestBody(required=True, content={media_type: OpenAPIMediaType(schema=schema)})
+    examples: Mapping[str, Example] | None = None
+    if isinstance(data_field.kwarg_definition, BodyKwarg) and isinstance(data_field.kwarg_definition.examples, list):
```

Comment:
```suggestion
    if isinstance(data_field.kwarg_definition, BodyKwarg) and data_field.kwarg_definition.examples is not None:
```

This check is a bit more resilient to refactorings than `isinstance(..., list)` (allow-list vs. block-list). With the suggested approach we could easily change the type of `KwargDefinition.examples` to allow any compatible type without having to adapt this check here.